packages/tests/tests/peer_exchange.node.spec.ts

```diff
@@ -37,7 +37,7 @@ describe("Peer Exchange", () => {
     waku = await createLightNode({
       libp2p: {
         peerDiscovery: [
-          bootstrap({ list: getPredefinedBootstrapNodes(Fleet.Test, 3) }),
+          bootstrap({ list: getPredefinedBootstrapNodes(Fleet.Prod, 3) }),
```

Comment:
Doesn't work with `Test` fleet.

Reply:
Does it work with test fleet **before** your changes?

Reply:
Actually, I think it was failing on the master (old versions too it seems), but since it is skipped in the CI we didn't notice it. 

I think it makes sense to this test run on CI.

Making an issue to investigate it - https://github.com/waku-org/js-waku/issues/1411

cc: @danisharora099

Reply:
Will be addressed in a separate PR.